openshift/template.yaml

```diff
@@ -10,7 +10,7 @@ parameters:
   required: true
 - name: BASE_DNS_DOMAINS # example: name1:id1/provider1,name2:id2/provider2
   value: ''
-- name: OPENSHIFT_INSTALL_RELEASE_IMAGE
+- name: OPENSHIFT_VERSIONS
```

Comment:
Is there a pr in gilab for this?

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/13328
Merged.